var/spack/repos/builtin/packages/gosam-contrib/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class GosamContrib(AutotoolsPackage):
+    """Additional libraries for GoSam MC generator"""
+
+    homepage = "https://gosam.hepforge.org"
+    url      = "https://gosam.hepforge.org/downloads/?f=gosam-contrib-2.0.tar.gz"
+
+    version('2.0', sha256='c05beceea74324eb51c1049773095e2cb0c09c8c909093ee913d8b0da659048d')
+    version('1.0', sha256='a29d4232d9190710246abc2ed97fdcd8790ce83580f56a360f3456b0377c40ec')
+
+    provides('oneloop')
+    provides('golem')
+    provides('ninja-mc')
+    provides('samurai')
```

Comment:
We don't normally use virtual dependencies unless there are multiple packages that can be used to fulfill the same dependency. If there's only a single package that provides `oneloop`, then you could instead use `depends_on('gosam-contrib')`.

Reply:
`gosam-contrib` is - technically speaking - a "bundle" type of package: [OneLoop](https://bitbucket.org/hameren/oneloop/src/master/) + [golem95](https://golem.hepforge.org/) + [ninja-mc](https://ninja.hepforge.org/) + [samurai](https://samurai.hepforge.org/)  (+ maybe others). The golem95 site is, however, missing from hepforge, and thus I couldn't provide it into as a separate package.

Reply:
I guess I would still wait until we have Spack packages for those other things before making it a virtual dependency.